include/IECoreScene/MeshAlgo.h

```diff
@@ -53,7 +53,21 @@ namespace MeshAlgo
 /// If provided, this will be periodically checked, and cancel the calculation with an exception
 /// if the canceller has been triggered ( indicating the result is no longer needed )
 
+// Enum for specifying how to weight normal calculation
+enum class NormalWeighting
+{
+	EqualVertex, // Equal weight to each vertex, simple, not very good
```

Comment:
`EqualVertex` sounds strange to me - I don't really know what the `Vertex` part is communicating. The obvious term to me would be "uniform", but maybe you're concerned about overlap with `Interpolation::Uniform`? Maybe just `Equal`?